autoswitch_virtualenv.plugin.zsh

```diff
@@ -26,22 +42,23 @@ function check_venv()
 
         SWITCH_TO=""
 
-        if [[ -f ".venv" ]]; then
-          file_owner="$(stat -c %u .venv)"
-          file_permissions="$(stat -c %a .venv)"
+        venv_path=$(check_venv_path "$PWD")
+        if [[ -n "$venv_path" ]]; then
```

Comment:
The empty string signals that the recursion traversed back to root and there was no .venv file. Checking whether the file exists happens in `check_venv_path`. There may be a more elegant way to do that. I'm open to suggestions.